### DISCUSSION TITLE
Component not displayed with ng-sidebar

### DISCUSSION THREAD

**Discussion by mmarchal** [npm/feedback#335] [General]
Hello, I have the igx-pie-chart component showing but when I add the ng-sidebar component, unable to display the chart in my page.
My component is in the <div ng-sidebar-content> and my sidebar is in the ng-sidebar
Do you have any idea what action I need to take to make it reappear?
Thanks a lot !

**Comment by MylesBorins** [comment]
Hey @mmarchal . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
